File: skills/astra-toolkit/clients/csharp/examples/admin-create-keyspace.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    // Get an existing database
    var client = new DataAPIClient();
    var database = client.GetDatabase(
      System.Environment.GetEnvironmentVariable("ASTRA_DB_API_ENDPOINT"),
      System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN")
    );

    var databaseAdmin = database.GetAdmin();

    await databaseAdmin.CreateKeyspaceAsync("**KEYSPACE_NAME**");
  }
}

File: skills/astra-toolkit/clients/typescript/examples/admin-create-keyspace.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// Get an admin object
const admin = database.admin();

// Create a keyspace
(async function () {
  await admin.createKeyspace("**KEYSPACE_NAME**");
})();
